src/NServiceBus.Core/Audit/AuditConfigReader.cs

```diff
@@ -2,17 +2,12 @@
 {
     using System;
     using System.Diagnostics;
-    using Config;
-    using Logging;
+    using NServiceBus.Config;
+    using NServiceBus.Logging;
```

Comment:
We need to tweak the R# setting to not qualify the NS, I will push a change to this branch